### ISSUE TITLE
Implement learning rate scheduler

### ISSUE THREAD

**Issue by luigibonati** [luigibonati/mlcvs#19]
LR scheduler is already in utils.optim but it is not implemented in NNCV base class

**Comment by luigibonati** [comment]
closed as switching to pytorch lightning scheduler
